python/aswfdocker/tests/test_builder.py

```diff
@@ -35,7 +35,7 @@ def test_package_baseqt_2019_dict(self):
         qt_version = list(
             index.Index().iter_versions(constants.ImageType.PACKAGE, "qt")
         )[0]
-        baked = b.make_bake_dict()
+        baked = b.make_bake_dict(False, False, False, False)
```

Comment:
I usually prefer defining default arg value on the function, and specify non-default args when needed using `my_arg=True, my_other_arg=True` as the `False, False, False, False` combo isn't very informative

Reply:
I was trying to go for "minimum footprint" changes and wasn't very clear on how best to propagate these parameters. I'll update the function signature accordingly.